src/sentry/db/models/manager.py

```diff
@@ -238,20 +238,24 @@ def create_or_update(self, **kwargs):
         return create_or_update(self.model, **kwargs)
 
     def bind_nodes(self, object_list, *node_names):
-        from sentry import app
+        from sentry.app import nodestore
 
         object_node_list = []
         for name in node_names:
-            object_node_list.extend((getattr(i, name) for i in object_list if getattr(i, name).id))
+            object_node_list.extend(
+                ((i, getattr(i, name))
```

Comment:
I know this pre-existed, but this would be more readable if `i` were `item` here, or `object`. Was a bit confusing to read since `i` is typically the index in a loop.